content/docs/concepts/asm/_index.md

```diff
@@ -0,0 +1,276 @@
+
+- unambiguously identify the CDP on-chain
+- by the CDP owner as the `msg.sender` address in all CDP interactions: to open, repay debt, and withdraw collateral from the CDP
+- by a Liquidator to identify the CDP being liquidated in a liquidation scenario.
+
+
+## Stabilization protocol
+
+ASM roles, core concepts, and the lifecycle of a CDP from opening to closure.
+
+### Stabilization roles
+
+There are three roles in the ASM:
+
+- *Borrower (CDP owner)*: a user taking out a CDP to borrow Auton, deposing collateral in return. There is no limit on the number of open CDP's a _borrower_ can own at any one time. Borrower's must have an [account](/glossary/#account) on the Autonity network.
+- *Liquidator (Keeper)*: a user or agent that liquidates a CDP that has brobecome under collateralized, repaying the CDP's outstanding debt and receiving remaining collateral token in return. Liquidator's must have an [account](/glossary/#account) on the Autonity network.
+- *Stabilizer (ASM Protocol)*: the `stabilizer` is the protocol account address used for Auton `mint` and `burn` operations by the Stabilization Contract)
+
+### ASM elements
+
+The ASM is composed of 3 system elements implemented as smart contract logic: ACU, Supply Control, Stabilization. ASM contracts are deployed by the protocol at genesis.
+
+#### ACU
+
+The [Auton Currency Unit (ACU)](/glossary/#acu) is a currency basket from which an index value is computed. This index value is then used as the _stabilization target_ for Auton price.
+
+The ACU currency basket is composed of 7 free-floating currencies:
+
+- AUD - Australian Dollar
+- CAD - Canadian Dollar
+- EUR - Euro
+- GBP - British Pound Sterling
+- JPY - Japanese Yen
+- SEK - Swedish Krona
+- USD - United States Dollar
+
+Each currency's quantity in the basket is computed to provide a currency basket with minimal variance. The index value then has minimal volatility with respect to variance from individual currency fluctuations.
+
+ACU is used as the _stabilization target_ for Auton price, see _[Stabilization](/concepts/asm/#stabilization)_.
+
+The value of ACU can be computed at any time in terms of exchange rates. ACU is computed based on the end-of-day time for the most recent 11 calendar years of data for each currency in the basket.
+
+{{% alert title="Info" %}}
+A 365 calendar day is used, end-of-day is 17:00 GMT. Price data is sourced from off-chain by the validator [oracle network](/concepts/oracle-network/) and retrieved from the [Oracle Contract](/concepts/architecture/#autonity-oracle-contract) on-chain.
+{{% /alert %}}
+
+USD is used as the numeraire over the data. The quantity of each currency in the basket is then computed based on a weighting that minimizes variance and using an initial target value of 1 USD for the total value of the basket. Basket quantities fixed over time. The value of SAC at any time can be computed in terms of exchange rates.
```

Comment:
USD is used as the numeraire over the data. The quantity of each currency in the basket is then computed based on a weighting that **aims to ~minimizes~ minimize its** variance **with respect to the basket** and using an initial target value of 1 USD for the total value of the basket. Basket quantities fixed over time. The value of SAC at any time can be computed in terms of exchange rates.

Reply:
I edited SAC to ACU.  

Else, done in commit 6240f03709feec9b14e779a078f0624f50b1aa7d